src/Phinx/Console/Command/Breakpoint.php

```diff
@@ -115,5 +115,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             // Toggle the breakpoint.
             $this->getManager()->toggleBreakpoint($environment, $version);
         }
+
+        return 0;
```

Comment:
You should use the class constants instead of magic numbers.

Reply:
Err, what class constant? The only ones I could find are
https://github.com/cakephp/phinx/blob/6b1c0edb507bb5b3ffb5056015a75b32052b6489/src/Phinx/Migration/Manager.php#L76-L84

I agree with using constants, can make a follow-up PR that makes those changes across all commands if desired (after decision on where those constants should live).

Reply:
`Command::CODE_SUCCESS`

Reply:
Yeah, there are no constants within Phinx or its current dependencies that can be used for this unfortunately, and I think doing it here for all commands would just muddle up this PR changeset.

Reply:
Within this issue would also be making more consistent for when an exception is raised or when to print an error and return exit code.

Reply:
https://github.com/cakephp/phinx/blob/master/src/Phinx/Console/Command/AbstractCommand.php#L47
Couldnt this just get 

    const CODE_SUCCESS = 0; 
    const CODE_ERROR = 1;

added?

Reply:
However, that then differs from how the codes are defined in Manager for status (they use a `EXIT_STATUS_` prefix). I think it would be best to unify everything in a separate PR after ironing out the details of #1623 then expanding the scope of this PR to do that, and for now just follow what `migrate` was doing which was already just returning magic numbers.